KiCAD 5.14 compatibility fix (py2&3)

easyw opened the pull request:
Fix compatibilty issues with:
KiCAD 5.1.4
Py2
Py3
Added button icon image

easyw committed:
KiCAD 5.14 compatibility fix (py2&3)

Fix compatibilty issues with:
KiCAD 5.1.4
Py2
Py3
Added button icon image

skuep commented on action_viafence/viafence_action.py:
```diff
@@ -89,7 +97,9 @@ def createVias(self, viaPoints, viaDrill, viaSize, netCode):
 
     def selfToMainDialog(self):
-        self.mainDlg.lstLayer.SetItems(self.layerMap.values())
+        #wx.LogMessage(str(self.layerMap.values()))
```
There seem to be some remaining comments from development/debugging I suppose.

easyw replied:
done with this commit
https://github.com/skuep/kicad-plugins/pull/8/commits/0ec66aee348139336e70d31d271eb7cc644a8695

skuep commented on action_viafence/viafence_action.py:
```diff
@@ -124,79 +134,90 @@ def mainDialogToSelf(self):
+        except:
+            wdlg = wx.MessageDialog(None, u"\u2718 ERROR Missing KiCAD \'pyclipper\' python module",'ERROR message',wx.OK | wx.ICON_ERROR)
+            result = wdlg.ShowModal()
+        if pyclip:
```
Why use the pyclip variable for this? This looks too complicated for what it does. 
Besides, why present a custom wxDialog with a custom message for a missing python dependency? 
I think python shows a meaningful message on its own, doesnt it?

easyw replied:
done with this commit
https://github.com/skuep/kicad-plugins/pull/8/commits/0ec66aee348139336e70d31d271eb7cc644a8695

skuep reviewed (changes requested):
Hi, Thanks for your pull request. 

The single commit in the pull request contains more than just changes for python2 compatibility. 
I don't think it is necessary or meaningful to add our own wxMessageDialog for reporting missing modules since Python outputs meaningful error messages itself. Besides we can just make pyclipper an "official" requirement in some part of the documentation.

easyw committed:
improved icon, dialog image, added fence for selected traces

easyw commented:
Hi @skuep ,
please test a working plugin, add at the top the code:
'import pyclipper' and run it without pyclipper in your system...
the plugin will just fail to load without any message. 
For that reason I've added this check ... to help users to debug a silent failure.  

Moreover ATM the KiCAD package distribution on win doesn't include pyclipper and the included pip doesn't allow to correctly install pyclipper on win.
But it is possible to just copy the corrected generated pyc lib on the bin dir of the win installation folder, so it is possible to easily let even win users to use this plugin.

easyw committed:
removing extra debug commented lines, removing extra pyclipper check

easyw commented:
@skuep I've removed extra commented debug code and I have removed also the pyclipper dialog and letting the script show the python exception (this was silent failing in the original version).

easyw committed:
avoiding a pcb crash in case of empty board

easyw committed:
adding Delete Vias button

easyw committed:
allowing ',' for decimal separator on inputs...

avoiding crash in case of input using ','
